Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenWindowControllerIOS.mm

```diff
@@ -894,6 +894,7 @@ - (void)enterFullScreen:(CGSize)videoDimensions
         [webView setAutoresizingMask:(UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight)];
         [webView setFrame:[_window bounds]];
         [webView _setMinimumEffectiveDeviceWidth:0];
+        [webView _setViewScale:1.f];
```

Comment:
@pxlcoder I think we only need to set the scale here. I see a similar issue on iOS.

I'm going to confirm if this fixes the issue on iOS too